<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registros</class>
 <widget class="QWidget" name="Registros">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>611</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>445</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cargando...</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/sigrdap/icons/historial.png</normaloff>:/sigrdap/icons/historial.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:enabled{
background-color: #3498db;
color: rgb(255, 255, 255);
}

QPushButton:disabled{
background-color: #adc1ce;
color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelValor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>18</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #2980b9;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>Valor</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDesconocido">
       <property name="toolTip">
        <string>Estado desconocido</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Estados/icons/unknownconnection.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSinConexion">
       <property name="toolTip">
        <string>Inactivo</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Estados/icons/noconnection.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelIntermitente">
       <property name="toolTip">
        <string>Intermitente</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Estados/icons/intermconnection.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelConectado">
       <property name="toolTip">
        <string>Activo</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Estados/icons/goodconnection.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="iconOn">
       <property name="toolTip">
        <string>Conectado</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/Estados/icons/ledon_small.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="iconOff">
       <property name="toolTip">
        <string>Desconectado</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/Estados/icons/ledoff_small.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="botonConfiguracion">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/sigrdap/icons/configsensor.png</normaloff>:/sigrdap/icons/configsensor.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tablaValores">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>430</width>
           <height>300</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">selection-color: #FFFFFF;
selection-background-color: #5bade4;
alternate-background-color: #ddf1ff;
</string>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string/>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QComboBox" name="seleccionarPeriodo">
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Última hora</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Día</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Personalizado</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QDateEdit" name="day">
             <property name="font">
              <font>
               <family>Verdana</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="displayFormat">
              <string>dd/MMM/yyyy</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="month">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>Verdana</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <item>
              <property name="text">
               <string>Enero</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Febrero</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Marzo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Abril</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mayo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Junio</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Julio</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Agosto</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Septiembre</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Octubre</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Noviembre</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Diciembre</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="year">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>Verdana</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="minimum">
              <number>1990</number>
             </property>
             <property name="maximum">
              <number>2199</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>2016</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="labelFechaInicial">
           <property name="font">
            <font>
             <family>Verdana</family>
            </font>
           </property>
           <property name="text">
            <string>Fecha inicial</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="fechaInicial">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="displayFormat">
            <string>dd/MMM/yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelFechaFinal">
           <property name="font">
            <font>
             <family>Verdana</family>
            </font>
           </property>
           <property name="text">
            <string>Fecha final</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="fechaFinal">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="displayFormat">
            <string>dd/MMM/yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="timeSpec">
            <enum>Qt::LocalTime</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="labelHoraInicial">
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Hora inicial</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="horaInicial">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="displayFormat">
            <string>HH:mm:ss</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="time">
            <time>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
            </time>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelHoraFinal">
           <property name="font">
            <font>
             <family>Verdana</family>
            </font>
           </property>
           <property name="text">
            <string>Hora final</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="horaFinal">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="displayFormat">
            <string>HH:mm:ss</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="botonGraficar">
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>GRAFICAR</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/VentanaHistorial/icons/grafica.png</normaloff>:/VentanaHistorial/icons/grafica.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="botonReportes">
           <property name="font">
            <font>
             <family>Verdana</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>REPORTES</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/VentanaHistorial/icons/calendario.png</normaloff>:/VentanaHistorial/icons/calendario.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="graficoBarra">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelGrupo">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(41, 128, 185);</string>
         </property>
         <property name="text">
          <string>Cargando...</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelTipo">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDireccion">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelFoto">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Imagen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
